source/Octopus.Tentacle/Kubernetes/PodLogLineParser.cs

```diff
@@ -62,23 +62,42 @@ static class PodLogLineParser
 
         public static PodLogLineParseResult ParseLine(string line, IPodLogEncryptionProvider encryptionProvider)
         {
-            var logParts = line.Split(new[] { '|' }, 4);
-            if (logParts.Length != 4)
+            var initialParts = line.Split(new[] { '|' }, 2);
+
+            var datePart = initialParts[0];
+
+            var remainingMessage = initialParts[1];
+
+            //get the first 2 control chars and check
+            var encryptionControl = remainingMessage.Substring(0, 2);
+
+            var isEncryptedMessage = false;
+            //there is an encryption control part at the start of the remaining message
+            if (encryptionControl.Equals("e|", StringComparison.Ordinal) ||
+                encryptionControl.Equals("p|", StringComparison.Ordinal))
```

Comment:
I'm leaning towards not adding in the `p` option, but will leave that with you

Reply:
yeah, fair

Reply:
Removed in [001d5ef](https://github.com/OctopusDeploy/OctopusTentacle/pull/1048/commits/001d5efd4c3d32630bc4dd2ad62f31e5195fcf7b) and updated test cases in [a1e5b8e](https://github.com/OctopusDeploy/OctopusTentacle/pull/1048/commits/a1e5b8eaf17d3f630a1375f6d9b389feabb92e67)